### ISSUE TITLE
[model:performance] Experiment with including bugs with their performance impact is `none` in the training set only

### ISSUE THREAD

**Issue by suhaibmujahid** [mozilla/bugbug#3901]
> Yeah, let's do that for now, if it is "none" it should not be considered as a perf bug.
>
> We can file a follow-up issue to investigate using these as performance or not as performance in the training set but excluding them from the test set to see if they have any impact.

_Originally posted by @marco-c in https://github.com/mozilla/bugbug/pull/3895#discussion_r1417400177_

Depends on: https://github.com/mozilla/bugbug/issues/3876

See also: https://github.com/mozilla/bugbug/issues/3923

**Comment by marco-c** [comment]
Depends on #4407.
